app/web/components/Button/Button.tsx

```diff
@@ -94,12 +92,6 @@ function _Button<D extends ElementType = "button">(
       variant={variant}
       color={variant === "contained" ? color : undefined}
     >
-      {(loading || waiting) && (
-        <CircularProgress
-          size={theme.typography.button.fontSize}
-          className={classes.spinner}
-        />
-      )}
```

Comment:
What's up with this one? Looks liked we removed it without adding anything back. I think since this is part of the button (and doesn't need to be centered), we can leave this one as is.

Reply:
I talked with Jesse about this and he agreed that the spinner in the button seemed a bit unnecessary since the button becomes greyed out and disabled during loading state, so the spinner just made it look more cluttered. I can add it back in though if needed!

Reply:
@graceegal The issue is this button is a reusable component all over the app, so unless we don't wan that spinner anywhere in the app, we shouldn't remove it.

If you check `AppButtonProps`, you can see the `loading` prop is optional. Better to just not pass that prop for the map button then, rather than remove it from all the buttons.

Reply:
Got it! I just added it back in to the button. Re the loading prop... if i take it out then it doesn't disable and grey out the button either. So I left the loading prop in so it now has the spinner in the button on the map but it looks okay!

Reply:
Try this for the button in `MapWrapper` to fix it:

```javascript
          <Button
            color="primary"
            disabled={isLoading}
            onClick={handleOnClick}
            className={classes.searchHereButton}
            endIcon={<ReplayIcon />}
          >
```

* remove loading prop
* add disabled prop and set it to `isLoading` to disable the button while loading.

Reply:
That worked! All updated now